packages/typespec-test/test/openai_non_branded/generated/typespec-ts/src/models/models.ts

```diff
@@ -210,42 +210,42 @@ export interface _CreateModerationResponseResultCategoryScores {
   /** The score for the category 'hate'. */
   hate: number;
   /** The score for the category 'hate/threatening'. */
-  "hate/threatening": number;
+  hateThreatening: number;
   /** The score for the category 'harassment'. */
   harassment: number;
   /** The score for the category 'harassment/threatening'. */
-  "harassment/threatening": number;
+  harassmentThreatening: number;
   /** The score for the category 'self-harm'. */
   selfHarm: number;
   /** The score for the category 'self-harm/intent'. */
-  "selfHarm/intent": number;
+  selfHarmIntent: number;
   /** The score for the category 'self-harm/instructive'. */
-  "selfHarm/instructive": number;
+  selfHarmInstructive: number;
   /** The score for the category 'sexual'. */
   sexual: number;
   /** The score for the category 'sexual/minors'. */
-  "sexual/minors": number;
+  sexualMinors: number;
   /** The score for the category 'violence'. */
   violence: number;
   /** The score for the category 'violence/graphic'. */
-  "violence/graphic": number;
+  violenceGraphic: number;
 }
 
 export function _createModerationResponseResultCategoryScoresDeserializer(
   item: any,
 ): _CreateModerationResponseResultCategoryScores {
   return {
     hate: item["hate"],
-    "hate/threatening": item["hate/threatening"],
+    hateThreatening: item["hate/threatening"],
```

Comment:
Please note the property name is changed because we remove the special char / here.

Reply:
i didn't notice we have dot cases. except / we have $ cases impacted also.